src/components/content/credential/AddCredential.tsx

```diff
@@ -0,0 +1,248 @@
+                ref={formRef}
+                form={form}
+                labelCol={{ span: 4 }}
+                wrapperCol={{ span: 14 }}
+                layout='horizontal'
+                style={{ maxWidth: 1000 }}
+                onFinish={submit}
+            >
+                <div className={'credential-from-input'}>
+                    <Form.Item label='Name' name='name' rules={[{ required: true }]}>
+                        <Input />
+                    </Form.Item>
+
+                    <Form.Item label='Csp' name='csp' rules={[{ required: true, message: 'Please Select Csp!' }]}>
+                        <Select onSelect={handleCspSelect}>
+                            {Object.values(CredentialVariables.csp).map((csp: CredentialVariables.csp) => {
+                                return (
+                                    <Select.Option key={csp} value={csp}>
+                                        {csp}
+                                    </Select.Option>
+                                );
+                            })}
+                        </Select>
+                    </Form.Item>
+                    <Form.Item
+                        label='Type'
+                        name='type'
+                        rules={[{ required: true, message: 'Please Select The Type of Credential!' }]}
+                    >
+                        <Select onSelect={handleCredentialTypeSelect}>
+                            {credentialTypeList.map((type: CredentialVariables.type) => {
+                                return (
+                                    <Select.Option key={type} value={type}>
+                                        {type}
+                                    </Select.Option>
+                                );
+                            })}
+                        </Select>
+                    </Form.Item>
+                    <Form.Item label='Description' name='description'>
+                        <TextArea rows={4} />
+                    </Form.Item>
+                    {/* eslint-disable-next-line @typescript-eslint/no-unnecessary-condition */}
+                    {credentialVariableList && credentialVariableList.length > 0 ? (
+                        <Form.Item
+                            label='Variables'
+                            name='variables'
+                            rules={[
+                                {
+                                    required: isContainsEmpty(credentialVariableList),
+                                    message: 'Please Input The Variables of Credential!',
+                                },
+                            ]}
+                        >
+                            <Table
+                                className={'add-credential-from-variables'}
+                                pagination={false}
+                                columns={columns}
+                                dataSource={credentialVariableList}
```

Comment:
still the mandatory variables are not displayed as mandatory. If you see here, none of the fields are marked as mandatory (with red star)

![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/8dd88be9-d3b8-42b1-9aae-7dcb5d342d13)